src/components/GlobalStyles.tsx

```diff
@@ -1,15 +1,41 @@
 import { css } from '@emotion/core'
-import { colorsV3, fonts, getCdnFontFaces } from '@hedviginsurance/brand'
+import { colorsV3, getCdnFontFaces } from '@hedviginsurance/brand'
 
 export const globalStyles = css`
   ${getCdnFontFaces()}
```

Comment:
Do we need these as well?

Reply:
Ah yes, when we get the from the brand package for sure 🙌